hera_stats/jackknives.py

```diff
@@ -519,7 +520,7 @@ def omit_ants(uvp, ant_nums):
         bl_i = np.array(map(uvp.antnums_to_bl, bl))[inds]
         uvp1 = uvp.select(bls=bl_i, inplace=False)
         uvp1.labels = np.array([ant_nums[i]])
-        uvp1.jkftype = "omit_ants"
+        uvp1.jktype = "omit_ants"
```

Comment:
I agree that a non-recognized class variable is shakey. What if I put it as an entry in stats_array? i.e. 
```
stats_array["jktype"][0] = np.array([jktype, jktype, jktype....])
```
where it would have the same shape as data_array (that's a requisite of stats_array stuff).